src/azure-cli/azure/cli/command_modules/policyinsights/_params.py

```diff
@@ -42,17 +42,17 @@ def load_arguments(self, _):
                 'namespace',
                 completer=get_providers_completion_list,
                 arg_group='Resource ID',
-                help='Provider namespace (Ex: ''Microsoft.Provider'').')
+                help='Provider namespace (Ex: Microsoft.Provider).')
```

Comment:
I think the author actually wants to quote `Microsoft.Provider`, but used the PowerShell syntax:

```
> echo "a""b"
a"b
> echo 'a''b'
a'b
```

```suggestion
                help="Provider namespace (Ex: 'Microsoft.Provider').")
```

Reply:
Should the following three help messages be kept consistent? Personally, I think when parentheses are used, having or not having double quotes is fine.